docs/content/preview/yugabyte-voyager/reference/data-migration/import-data.md

```diff
@@ -93,14 +126,126 @@ The valid *arguments* for import data status are described in the following tabl
+| --source-replica-db-name <name> | Name of the database in the source-replica database server on which import needs to be done. |
+| --source-replica-db-password <password> | Password to connect to the source-replica database server. Alternatively, you can also specify the password by setting the environment variable `SOURCE_REPLICA_DB_PASSWORD`. If you don't provide a password via the CLI or environment variable during any migration phase, yb-voyager will prompt you at runtime for a password. If the password contains special characters that are interpreted by the shell (for example, # and $), enclose the password in single quotes. |
+| --source-replica-db-port <port> | Port number of the source-replica database server. <br>Default: 1521 (Oracle) |
+| --source-replica-db-schema <schemaName> | Schema name of the source-replica database. |
+| --source-replica-db-sid <SID> | Oracle System Identifier (SID) that you wish to use while importing data to Oracle instances. Oracle migrations only. |
+| --source-replica-db-user <username>| Username to connect to the source-replica database server. |
+| --source-replica-db-ssl-cert <path>| Source-replica database SSL Certificate path. |
+| --source-replica-db-ssl-crl <list>| Source-replica database SSL Root Certificate Revocation List (CRL). |
+| --source-replica-db-ssl-key <Keypath> | Source-replica database SSL key path. |
+| --source-replica-db-ssl-mode <SSLmode>| One of `disable`, `allow`, `prefer`(default), `require`, `verify-ca`, or `verify-full`. |
+| --source-replica-db-ssl-root-cert | Source-replica database SSL Root Certificate path. |
+| -h, --help | Command line help for import data to source-replica. |
+| --oracle-home <path> | Path to set $ORACLE_HOME environment variable. `tnsnames.ora` is found in `$ORACLE_HOME/network/admin`. Oracle migrations only.|
+| [--oracle-tns-alias](../../yb-voyager-cli/#ssl-connectivity) <alias> | TNS (Transparent Network Substrate) alias configured to establish a secure connection with the server. Oracle migrations only. |
+| --parallel-jobs <connectionCount> | The number of parallel batches issued to the source-replica database. <br> Default: 1 |
+| --start-clean | Starts a fresh import with data files present in the `data` directory.<br>If there's any non-empty table on the target YugabyteDB database, you get a prompt whether to continue the import without truncating those tables; if you go ahead without truncating, then yb-voyager starts ingesting the data present in the data files with upsert mode.<br> **Note** that for cases where a table doesn't have a primary key, it may lead to insertion of duplicate data. In that case, you can avoid the duplication by excluding the table from the `--exclude-table-list`, or truncating those tables manually before using the `start-clean` flag. <br> Accepted parameters: true, false, yes, no, 0, 1 |
+
```

Comment:
rename these (for SSL ones `db` in argument name is not there) -
```
--source-replica-db-ssl-cert  -> --source-replica-ssl-cert 
--source-replica-db-ssl-crl -> --source-replica-ssl-crl 
--source-replica-db-ssl-key ->--source-replica-ssl-key 
--source-replica-db-ssl-mode ->--source-replica-ssl-mode 
--source-replica-db-ssl-root-cert ->--source-replica-ssl-root-cert 
```

add two more arguments - 
```
| --send-diagnostics | Enable or disable sending [diagnostics](../../../diagnostics-report/) information to Yugabyte. <br>Default: true<br> Accepted parameters: true, false, yes, no, 0, 1 |
| -y, --yes | Answer yes to all prompts during the migration. <br>Default: false |
```